include/bluetooth/bluetooth.h

```diff
@@ -1213,7 +1228,110 @@ int bt_le_per_adv_sync_recv_enable(struct bt_le_per_adv_sync *per_adv_sync);
  *
  * @return Zero on success or (negative) error code otherwise.
  */
+
 int bt_le_per_adv_sync_recv_disable(struct bt_le_per_adv_sync *per_adv_sync);
+/** Periodic Advertising options */
+enum {
+	/** Convenience value when no options are specified. */
+	BT_LE_PAST_OPT_NONE = 0,
+
+	/** Sync with Angle of Arrival (AoA) constant tone extension */
+	BT_LE_PAST_OPT_DONT_SYNC_AOA = BIT(0),
```

Comment:
The DONT in the name does not match up with the comment above. I think NO would be better than DONT (and shorter).

Reply:
I agree :)